index.html

```diff
@@ -6,14 +6,324 @@
+    <section class="middle-page mt-5 pt-5 mx-lg-5 text-center">
+        <article class="row g-0">
+            <h2 class="col-12">About</h2>
+            <h3 class="col-12">Maria Hope</h3>
+            <p class="col-8 offset-2 mt-4">I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me toadd your own content and make changes to the font. Feel free to drag and drop me anywhere you like on your page. I’m agreat place for you to tell a story and let your users know a little more about you. </p>
+            <a href="#">
+                <svg class="right-icon" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-arrow-right" viewBox="0 0 16 16">
+                    <path fill-rule="evenodd" d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8z" />
+                </svg>
+            </a>
+        </article>
+        
+        <figure> 
+            <img src="./img/9c608a_2a78cf5937414d0e9a597504d68daafc_mv2_d_6046_4035_s_4_2.webp" alt="Picture Of Maria Hope"> 
+        </figure>
+    </section>
+                         
+                        <!-- Adding the bottom part of the page (Services such as "Individual Therapy" etc.) -->
+    <section class="bottom-page mx-lg-5 mt-lg-5 pt-lg-5">
+        <article class="row g-0 text-center ">
+            
+            <article class="article-one px-5 col-12 col-lg-4 d-flex flex-column justify-content-center ">
+                <h5>01.</h5>
+                <a href="#">Individual Therapy</a>
+                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Labore obcaecati possimus sit iusto nobis ad dolore repudiandae doloribus culpa facilis!</p>
+            </article>
+            
+            <article class="article-two px-5  col-12 col-lg-4 d-flex flex-column justify-content-center align-items-center">
+                <h5>02.</h5>
+                <a href="#">Group Therapy</a>
+                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Labore obcaecati possimus sit iusto nobis ad dolore repudiandae doloribus culpa facilis!</p>
+                <a class="view-more" href="#" >View More Services</a>
+            </article>
+            
+            <article class="article-three px-5 col-12 col-lg-4 d-flex flex-column justify-content-center">
+                <h5>03.</h5>
+                <a href="#">Couples Therapy</a>
+                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Labore obcaecati possimus sit iusto nobis ad dolore repudiandae doloribus culpa facilis!</p>
+            </article>
+        
+        </article>
+    </section>
+                       
+                        <!-- Adding a Div to act as a space between two sections -->
+    <div class="space" style="height: 280px;"></div>
```

Comment:
I think you should try to use CSS to add whitespace between the sections instead of adding this `div` in. Remember, you want to try and have all your style related code in CSS, not in the HTML.

Reply:
thanks for your insight, I will follow this rule from now on.